tests/framework/e2e/cluster.go

```diff
@@ -177,17 +179,14 @@ type EtcdProcessClusterConfig struct {
 }
 
 func DefaultConfig() *EtcdProcessClusterConfig {
-	return &EtcdProcessClusterConfig{
+	cfg := &EtcdProcessClusterConfig{
 		ClusterSize: 3,
 		CN:          true,
 
-		ServerConfig: embed.Config{
-			InitialClusterToken:    "new",
-			StrictReconfigCheck:    true,
-			SnapshotCount:          etcdserver.DefaultSnapshotCount,
-			SnapshotCatchUpEntries: etcdserver.DefaultSnapshotCatchUpEntries,
-		},
+		ServerConfig: *embed.NewConfig(),
 	}
+	cfg.ServerConfig.InitialClusterToken = "new"
```

Comment:
Note, there are tests that depend on this value being exact "new". I might do a followup PR to remove this.